src/apps/chifra/pkg/account/account.go

```diff
@@ -0,0 +1,35 @@
+package account
```

Comment:
I'm not a huge fan of having a package called `account`. I've struggled with this, and the distinction between `account` and `address` gets really fuzzy. Can we put this into the `rpcClient` package? In fact, it may already be in there for other reasons.

Reply:
Sure

Reply:
I cannot put it in `rpcClient`, because of circular dependencies, but I'll rename it to `contract` instead